src/components/Repository/Repository.tsx

```diff
@@ -11,47 +10,36 @@ import List from '../../muiComponents/List';
 import git from './img/git.png';
 import { GithubLink, StyledText, RepositoryListItem, RepositoryListItemText } from './styles';
 
-class Repository extends Component {
-  public render(): ReactElement<HTMLElement> {
-    return (
-      <DetailContextConsumer>
-        {context => {
-          return context && context.packageMeta && this.renderRepository(context.packageMeta);
-        }}
-      </DetailContextConsumer>
-    );
-  }
-
-  private renderRepositoryText(url: string): ReactElement<HTMLElement> {
-    return (
-      <GithubLink href={url} target="_blank">
-        {url}
-      </GithubLink>
-    );
-  }
-
-  private renderRepository = packageMeta => {
-    const { repository: { url = null } = {} } = packageMeta.latest;
-
-    if (!url || isURL(url) === false) {
-      return null;
-    }
-
-    return (
-      <Fragment>
-        <List dense={true} subheader={<StyledText variant="subtitle1">{'Repository'}</StyledText>}>
-          <RepositoryListItem button={true}>
-            <Avatar src={git} />
-            <RepositoryListItemText primary={this.renderContent(url)} />
-          </RepositoryListItem>
-        </List>
-      </Fragment>
-    );
-  };
-
-  private renderContent(url: string): ReactElement<HTMLElement> {
-    return <CopyToClipBoard text={url}>{this.renderRepositoryText(url)}</CopyToClipBoard>;
-  }
-}
+const Repository: React.FC = props => {
+	const renderRepositoryText = (url: string) => (
+		<GithubLink href={url} target="_blank">
+			{url}
+		</GithubLink>
+	);
+
+	const renderContent = (url: string) => <CopyToClipBoard text={url}>{renderRepositoryText(url)}</CopyToClipBoard>;
```

Comment:
IMO you could ignore by adding `/* eslint-disable react/jsx-max-depth */` it doesn't make sense to keep such a little piece of code separate like this

Reply:
Agree, in this case makes totally sense what @priscilawebdev suggest.